local-cli/runIOS/findMatchingSimulator.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.*
+ */
+'use strict';
+
+function findMatchingSimulator(simulators, simulatorName) {
+  if (!simulators.devices) {
+    return null;
+  }
+
+  const devices = simulators.devices;
+
+  var match;
+
+  for (let version in devices) {
+    // Making sure the version of the simulator is an iOS (Removes Apple Watch, etc)
+    if(version.indexOf('iOS') !== 0){
+      continue;
+    }
+
+    for (let i in devices[version]) {
+      let simulator = devices[version][i];
+
+      // Skipping non-available simulator
+      if(simulator.availability != "(available)"){
+        continue;
+      }
+
+      // If there is a booted simulator, we'll use that as instruments will not boot a second simulator
+      if(simulator.state == "Booted"){
```

Comment:
nit: space after `)` (everywhere, for consistency with the rest of this file and the codebase)